### PR TITLE
147 getting hands on

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/147)
[staging](https://650cd323e2ef45009613f05d--tangerine-buttercream-20c32f.netlify.app/developers/academy/zero_to_mvp/hello_weaviate/hands_on)

### What's being changed:
updates for the hands on page

### Type of change:
- [X] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [X] **Github action** – automated build completed without errors
- [X] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

**Comment by daveatweaviate** [comment]
@databyjp would you have a look at this one when you get a moment? Thanks!

File: developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx [hunk 1]
```diff
@@ -2,13 +2,13 @@
 title: Getting hands-on
 ---
 
-### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;Preparation
+### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp; Preparation
 
 :::note Obtain a free trial OpenAI API key
-This section includes queries using the OpenAI inference endpoint. So, we recommend obtaining an OpenAI account, and obtaining a **free API key which should be sufficient** for all of the below.
+This section includes queries using the OpenAI inference endpoint. If you don't already have an OpenAI account, we recommend getting an account with a **free API key** The free key should be sufficient for these exercises.
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx:8]: Hmm. I do not think they do free keys any more unfortunately.

Let's just rephase it as follows:

`This section includes queries using an OpenAI inference endpoint. If you don't already have an OpenAI account, we recommend creating one. At the time of writing, OpenAI provides trial credits which should be sufficient for these exercises.`

File: developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx [hunk 2]
```diff
@@ -17,72 +17,86 @@ import ReactPlayer from 'react-player/lazy'
 
 Now that you've set up your own Weaviate instance and installed a client, let's get hands-on with Weaviate.
 
-### <i class="fa-solid fa-code"></i>&nbsp;&nbsp;Client instantiation
+### <i class="fa-solid fa-arrow-right-from-bracket"></i>&nbsp;&nbsp; Client instantiation
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx:21]: Hey, I think I prefer the code symbol to distinguish between the theory and practical symbols. Can we leave it for now?

File: developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx (hunk 1 above)
Comment (daveatweaviate) [reply developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx]: Bummer! 

Updated

File: developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx (hunk 2 above)
Comment (daveatweaviate) [reply developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx:21]: Updated. 

I'll update the other PR(s) after you've had a look at them.

**Review by daveatweaviate — COMMENTED** [review]
@databyjp thanks for your feedback, this is ready for another look

[updated staging](https://650debd4fc0bfc2c0ab8aca8--tangerine-buttercream-20c32f.netlify.app/developers/academy/zero_to_mvp/hello_weaviate/hands_on)

File: developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx [hunk 3]
```diff
@@ -17,72 +17,86 @@ import ReactPlayer from 'react-player/lazy'
...
 Now that you've set up your own Weaviate instance and installed a client, let's get hands-on with Weaviate.
 
-### <i class="fa-solid fa-code"></i>&nbsp;&nbsp;Client instantiation
+### <i class="fa-solid fa-code"></i>&nbsp;&nbsp; Client instantiation
 
-A `client` object can be instantiated for convenient access to your instance of Weaviate. You can set the following parameters:
+Create a `client` object for persistent access to your Weaviate instance. You can set the following parameters:
 
-#### <i class="fa-solid fa-code-simple"></i> Host URL (required):
+- **Host URL (required)**
```
[the remark this answers is not in the document]
Comment (daveatweaviate) [reply developers/academy/zero_to_mvp/101_hello_weaviate/50_hands_on.mdx:24]: They are all parameters under the "Client instantiation" heading. I thought the bold text made them look more like a single unit. It's probably a good candidate for updating to a table as well.

**Pull request merged**